pom.xml

```diff
@@ -339,7 +341,7 @@
                     </dependencies>
                     <configuration>
                         <forkNode implementation="org.apache.maven.plugin.surefire.extensions.SurefireForkNodeFactory"/>
-                        <redirectTestOutputToFile>true</redirectTestOutputToFile>
+                        <redirectTestOutputToFile>${maven.test.redirectTestOutputToFile}</redirectTestOutputToFile>
```

Comment:
`maven.test.redirectTestOutputToFile` is the user property for `redirectTestOutputToFile`. Just out of curiosity @jamezp , it didn't work well by having it defined at the property section above in this `pom.xml`?

Without playing with it, looks like it could be optional to define it here

Reply:
You're right Yeray. That works, I'll update the PR now.